test/cloudnative/cloudnative_test.go

```diff
@@ -21,11 +22,13 @@ func TestMain(m *testing.M) {
 	testEnvironment.BeforeEachTest(dynakube.DeleteIfExists(dynakube.NewBuilder().WithDefaultObjectMeta().Build()))
 	testEnvironment.BeforeEachTest(oneagent.WaitForDaemonSetPodsDeletion())
 	testEnvironment.BeforeEachTest(namespace.Recreate(namespace.NewBuilder(dynakube.Namespace).Build()))
+	testEnvironment.BeforeEachTest(proxy.DeleteProxyIfExists())
```

Comment:
is this still needed ?

Reply:
thank you for pointing out, forgot about it :facepalm: